test/test_discordmessageprocessor.ts

```diff
@@ -537,6 +537,38 @@ describe("DiscordMessageProcessor", () => {
 TestDescription`,
             );
         });
+        it("adds fields properly", () => {
+            const processor = new DiscordMessageProcessor(
+                new DiscordMessageProcessorOpts("localhost"), bot as DiscordBot);
+            const msg = new MockMessage() as any;
+            msg.embeds = [
+                new Discord.MessageEmbed(msg, {
+                    description: "TestDescription",
+                    title: "TestTitle",
+                    url: "testurl",
+                }),
+            ];
+            msg.embeds[0].fields = [
+                {
+                    embed: msg.embeds[0],
+                    inline: false,
+                    name: "fox",
+                    value: "floof",
+                },
+            ] as any;
+            const inContent = "Content that goes in the message";
+            const content = processor.InsertEmbeds(inContent, msg);
+            Chai.assert.equal(
+                content,
+`Content that goes in the message
+
+----
+##### [TestTitle](testurl)
+TestDescription
+**fox**
```

Comment:
Hmm, do we want these as a table, or at least next to each other rather than on newlines. E.g. `**fox**: floof` ?

Reply:
discord actually has an option for that, the `inline` property https://discord.js.org/#/docs/main/stable/class/MessageEmbedField

For simplicity soru assumed that inline is false

Reply:
Can haz screenshot? :3

Reply:
Looks a bit silly because silly testing bot. No actual message content, thus the horizontal bar as start of embed

![Screenshot_2019-04-23_21-20-01](https://user-images.githubusercontent.com/2433620/56609442-6795f680-660d-11e9-81a3-fedabf0f3ae1.png)

Reply:
I guess that works :)